lib/Minz/Request.php

```diff
@@ -294,8 +348,41 @@ private static function magicQuotesOff() {
 		}
 	}
 
+	/**
+	 * @return array
+	 */
+	private static function getHeaders() {
```

Comment:
What is the purpose of this function? Is seems to just make a manual copy of `$_SERVER`

Reply:
I've extracted that in case we need to pre-process some of the keys or values. At the moment, it has no extra values than storing locally the variable ```$_SERVER```